src/basic/architecture.h

```diff
@@ -45,7 +45,7 @@ enum {
         ARCHITECTURE_ARC,
         ARCHITECTURE_ARC_BE,
         _ARCHITECTURE_MAX,
-        _ARCHITECTURE_INVALID = -1
+        _ARCHITECTURE_INVALID = -EINVAL,
```

Comment:
hmm, I think you can change the 3 cases of this in src/systemd/ too. they are in sd-lldp.h + sd-dhcp-lease.h + sd-ndisc.h, and none of those APIs we currently declared official API and install to includedir, iirc?

Reply:
OK.